src/config.js

```diff
@@ -178,6 +178,14 @@ export const WMS_TILE_SIZE = 512 // px
 
 export const VIEW_MIN_RESOLUTION = 0.1 // meters/pixel
 
+/**
+ * Map default max resolution as it is set in mf-chsdi. When the layer has this as a max resolution,
+ * the layersConfig endpoint does not give the resolutions available.
+ * https://github.com/geoadmin/mf-chsdi3/blob/0236814544a6bf2df86598889b81ee4023494325/chsdi/models/bod.py#L119-L123
+ *
+ * @type {Number}
+ */
+export const DEFAULT_MAX_GEOADMIN_RESOLUTION = 0.5 // meters/pixel
```

Comment:
I would maybe switch words GEOADMIN and MAX here, and add WMTS somewhere.
Something like `DEFAULT_GEOADMIN_MAX_WMTS_RESOLUTION` (as we now know it is only applicable to WMTS)